app/workers/hard_worker.rb

```diff
@@ -0,0 +1,9 @@
+class HardWorker
+  include Sidekiq::Worker
+
+  def perform(current_user_email, total_users, total_articles)
+    puts 'Doing hard work'
+    puts current_user_email
+    ArticleMailer.email_last_users_articles(current_user_email,5,10).deliver
```

Comment:
Usually when you are in a worker and an email is sent. People use deliver! to avoid the sidekiq queue to get bigger and bigger